pennylane/qchem/factorization.py

```diff
@@ -394,6 +394,7 @@ def _chemist_transform(one_body_tensor=None, two_body_tensor=None, spatial_basis
 
     if two_body_tensor is not None:
         chemist_two_body_coeffs = np.swapaxes(two_body_tensor, 1, 3)
+        # pylint:disable=invalid-unary-operand-type
```

Comment:
why weren't we requiring it before?

Reply:
my guess is that pylint couldn't inspect the call signature of `pennylane.numpy.einsum` because it's a meta-module, but i'm not sure

Reply:
Could be, nice catch!